packages/terra-docs/src/Docs.jsx

```diff
@@ -0,0 +1,31 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import classNames from 'classnames/bind';
+import styles from './Docs.module.scss';
+
+const cx = classNames.bind(styles);
+
+const propTypes = {
+  /**
+   * Content to be displayed as the name
+   */
+  name: PropTypes.string,
+};
+
+const defaultProps = {
+  name: 'default',
```

Comment:
Based on PR description, Is this just a template at this point?

Reply:
yes